package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "appwrite",
+  "homepage": "https://appwrite.io/support",
+  "description": "Appwrite backend as a service cuts up to 70% of the time and costs required for building a modern application. We abstract and simplify common development tasks behind a REST APIs, to help you develop your app in a fast and secure way. For full API documentation and tutorials go to [https://appwrite.io/docs](https://appwrite.io/docs)",
+  "version": "",
+  "license": "BSD-3-Clause",
+  "main": "index.ts",
```

Comment:
1.  Libraries usually expose JavaScript, not TypeScript, you don't want to force your customers to use TypeScript. Transpile your code and expose the equivalent`.js` file here.
2. You might want to generate both `cjs` and `esm` versions of your library, so consumers could choose the best for them. The `esm` entry should be exposed via the `module` property in `package.json`.
3. Also, adding `"sideEffects": true` in addition to the `ems` would make your library tree-shakeable, which is super useful and widely supported and expected this days.